test/helpers/extended_timeout_helper.rb

```diff
@@ -0,0 +1,25 @@
+module ExtendedTimeoutHelper
+  DEFAULT_EXTENDED_WAIT_TIME = 5
+
+  def self.included(base)
+    base.setup :increase_wait_time
+    base.teardown :restore_wait_time
+  end
+
+  def with_wait_time(seconds)
+    @wait_time_override = seconds
+  end
```

Comment:
This has been added to for the sake of configurability, but it's not used anywhere. Happy to make it go away if you think YAGNI applies here 😄